packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -166,6 +181,35 @@ export function countOperationsMutation(oas: Oas3): number {
   return numOps
 }
 
+/**
+ * Counts the number of operations that translate to subscriptions in an OAS.
+ */
+export function countOperationsSubscription(oas: Oas3): number {
+  let numOps = 0
+  for (let path in oas.paths) {
+    for (let method in oas.paths[path]) {
+      if (
+        isOperation(method) &&
+        method.toLowerCase() !== 'get' &&
+        oas.paths[path][method].callbacks
+      ) {
+        for (let cbName in oas.paths[path][method].callbacks) {
+          for (let cbPath in oas.paths[path][method].callbacks[cbName]) {
+            numOps++
+            // resolve $ref ??
```

Comment:
Still unsure if we should count deeper, each path contained in a callback as distinct callback

Reply:
I'm not really sure either. It depends on the use cases. Do we expect people to have multiple operation objects on the same callback object? I'm guessing no... If that is the case, then we not count deeper. 

We should however ensure that an operation object exists. Let's just assume that there can only be one and throw a warning if multiple ones are found.

Reply:
It makes echo to the comment above on `preprocessor.ts`
Maybe counting one operation would be easier to start with and we could slowly iterate to count and handle properly all of them later.